tests/installation/releasenotes.pm

```diff
@@ -77,7 +77,7 @@ sub run {
         if (!check_var('VIDEOMODE', 'text')) {
             # Make sure release notes window is shown to avoid sending key too early
             # It takes longer time to show multilple release notes for addons
-            assert_screen([qw(release-notes-sle-ok-button release-notes-sle-close-button)], 120);
+            assert_screen([qw(release-notes-sle-ok-button release-notes-sle-close-button)], get_var('UPGRADE') ? 300 : 120);
```

Comment:
Why should this screen take longer to load on upgrade?

Reply:
Ha, actually the change here with longer timeout is not related to live-patching, but for another issue observed in migration with multiple addons, such as:
https://openqa.suse.de/tests/1540517#step/releasenotes/25
https://openqa.suse.de/tests/1540856#step/releasenotes/4

During upgrade, release notes window takes longer time to show after pressing "release notes button", because machine is busy with "Analyzing your system" at background. Even much longer time is required when upgrade with all-addons and all-patterns, and even worse when openQA-worker is loaded. It doesn't look like a product bug here, since it sounds reasonable to take some time to analyze the system in upgrade.

At worst case, it took ~230s to show the release notes window (with all-addons), for example following case debugged with TIMEOUT_SCALE=4
https://openqa.suse.de/tests/1541844#step/releasenotes/2

I know I'd better to submit another PR to change the timeout, sorry for the confusing caused here.

Reply:
@okurz Need I submit another PR for the timeout change?

Reply:
You can either just change this PR or create a new one.

Reply:
Just updated this PR (replace 120s timeout with 300s to make sure release notes window is shown)